lua/org-bullets.lua

```diff
@@ -18,12 +18,13 @@ local list_groups = {
 local defaults = {
   show_current_line = false,
   symbols = {
-    headlines = { "◉", "○", "✸", "✿" },
+    headlines = { " ", " ", " " },
```

Comment:
@lvim-tech please leave all the symbols as is. I took these from the original emacs project because I like them. A user can change them if they want but they should be the default